src/core/dev/net_device_val.cpp

```diff
@@ -70,8 +70,12 @@
 #define nd_logfunc    __log_info_func
 #define nd_logfuncall __log_info_funcall
 
+//This constructor is called to initialize class members of type ring_alloc_logic_attr that are not
```

Comment:
The first 3 lines are obvious and not needed.
Its enough to say that the value given in this constructor is later always overridden.